### PR TITLE
[#2] feature/2

### PR DESCRIPTION
**Pull request by sjhello**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -0,0 +1 @@
+이슈 2번
```
Comment (sjhello) [inline README.MD:1]: zxczxc

Comment (a656963) [inline README.MD:1]: aaaa

**Review by a656963 — APPROVED** [review]

File: README.MD (hunk 1 above)
Comment (a656963) [reply README.MD:1]: sdfsdf

**Review by a656963 — CHANGES_REQUESTED** [review]

File: README.MD (hunk 1 above)
Comment (a656963) [inline README.MD:1]: dfsdfsfdsdf

**Review by a656963 — APPROVED** [review]

**Pull request merged**